### PR TITLE
AboutSection was edited with tailwindcss

### PR DESCRIPTION
**Pull request by min-in-zon**
#15

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ export default {
           '2xl': '183px',
         },
       },
+      fontSize: {
+        '2rem': '2rem',
```
Comment (adadychanka) [inline tailwind.config.js:21]: Why?

**Review by adadychanka — APPROVED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (adadychanka) [reply tailwind.config.js:21]: I checked the following PRs, are you going to use it as text-2rem?

Comment (min-in-zon) [reply tailwind.config.js:21]: Yes, TW doesn't have 2rem. And I have many elements with 2rem and it's really look better with 2rem (not 1.1875 ot 2.25rem). 
Or do u think that name 2rem is not a good idea?
image

Comment (adadychanka) [reply tailwind.config.js:21]: Let's leave it as it is

**Review by adadychanka — APPROVED** [review]

**Comment by min-in-zon** [comment]
I fixed merge conflict in AboutSection, w8 for approve

**Review by adadychanka — APPROVED** [review]

**Pull request merged**